frontend/src/app/features/folios/AddToFolio.tsx

```diff
@@ -26,14 +28,24 @@ const AddToFolio: FC<AddToFolioProps> = ({
   selectedSiteIds,
   popupPlacement = 'bottom-start',
   triggerClassName,
+  showSearchBoxOnly = false,
+  handleClose
 }) => {
   const [searchTerm, setSearchTerm] = useState('');
 
+  handleClose = handleClose ??( (event:any)=>{});
+
   const auth = useAuth();
 
   const [getFolioItems, { loading, data }] =
     useFolio_GetFolioItemsForUserLazyQuery();
 
+   useEffect(()=>{
+    if (showSearchBoxOnly) {
+      getFolioItems();
+    }
+   }, []) 
```

Comment:
This `useEffect` should have dependencies:

```suggestion
   useEffect(() => {
    if (showSearchBoxOnly) {
      getFolioItems();
    }
  }, [getFolioItems, showSearchBoxOnly]);
```

Reply:
removed as no longer needed with updated code